Cargo.toml

```diff
@@ -15,14 +15,14 @@ members = [
 ]
 
 [dependencies]
-secp256k1 = "0.26"
+secp256k1 = "0.28.0"
 rand_core = "0.6.4"
-ring = "0.16"
+ring = "0.17.5"
 lazy_static = "1.4"
-thiserror = "1.0.38"
+thiserror = "1.0.50"
 
 [dev-dependencies]
-hex = "0.4"
-base58 = "0.1"
-ripemd160 = "0.8"
```

Comment:
According to its crates.io entry, `[ripemd160](https://crates.io/crates/ripemd160)` is deprecated and should be replaced with `[ripemd](https://crates.io/crates/ripemd)`.